pangeo-binder/values.yaml

```diff
@@ -64,7 +72,26 @@ binderhub:
         limits:
           cpu: "1.25"
           memory: 1Gi
+      allowNamedServers: true
+      # change this value as you wish,
+      # or remove this line if you don't want to have any limit
+      # namedServerLimitPerUser: 5
+      redirectToServer: false
+      services:
+        binder:
+          oauth_redirect_uri: "https://hub.staging.binder.pangeo.io/hub/oauth_callback"
```

Comment:
"https://hub.staging.binder.pangeo.io/hub/oauth_callback" -> I think this should be url of the binderhub: "https://staging.binder.pangeo.io/hub/oauth_callback"